DiceKeys/Views/Screens/EditableDiceKeyView/DiceKeyboardView.swift

```diff
@@ -55,6 +55,14 @@ struct DiceKeyboardView: View {
     }
 
     var body: some View {
+        #if os(macOS)
```

Comment:
ios devices (especially pro) can have keyboards, so at a minimum the onReceive event should exist for all operating systems, right?

Reply:
onReceive only for macOS.